maat-scheduled-tasks/src/test/resources/application.yaml

```diff
@@ -21,6 +14,13 @@ spring:
     properties:
       hibernate.enable_lazy_load_no_trans: true
       hibernate.jdbc.lob.non_contextual_creation: true
+  datasource:
+    url: jdbc:h2:mem:mla;MODE=Oracle;DB_CLOSE_DELAY=-1;IGNORE_CATALOGS=TRUE;NON_KEYWORDS=VALUE;INIT=CREATE SCHEMA IF NOT EXISTS togdata\;CREATE SCHEMA IF NOT EXISTS hub\;SET SCHEMA togdata
```

Comment:
The Hub schema is defined in the s`chema-h2.sql` file so we don't need to add it here.